topics/chaos-engineering/index.md

```diff
@@ -0,0 +1,10 @@
+---
+aliases: chaos-engineering, site-reliability-engineering
```

Comment:
this alias list cannot include the topic name
```suggestion
aliases: site-reliability-engineering
```

Reply:
sure, i will make this change @kenyonj